src/pages/config/config-details-changes.jsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState } from "react";
+import { useParams, useOutletContext } from "react-router-dom";
+
+import { getConfigChange, getConfig } from "../../api/services/configs";
+import { toastError } from "../../components/Toast/toast";
+import { ConfigChangeHistory } from "../../components/ConfigChangeHistory";
+import { BreadcrumbNav } from "../../components/BreadcrumbNav";
+
+export function ConfigDetailsChangesPage() {
+  const { id } = useParams();
+  const [isLoading, setIsLoading] = useState(true);
+  const [historyData, setHistoryData] = useState([]);
+  const { setTitle } = useOutletContext();
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getConfigChange(id)
+      .then((res) => {
+        if (res.data.length === 0) {
+          setHistoryData([]);
+        } else {
+          setHistoryData(res?.data);
+        }
+      })
+      .catch((err) => toastError(err))
+      .finally(() => {
+        setIsLoading(false);
+      });
+
+    getConfig(id)
+      .then((res) => {
+        const data = res?.data[0];
+        setTitle(
+          <BreadcrumbNav
```

Comment:
inline literals introduces unnecessary react dom reconciliations. 
Can we use memoized constants here.